### PR TITLE
Refactor wrapper (and make compileJSON optional)

### PR DESCRIPTION
**Pull request by axic**
Part of #120 and https://github.com/ethereum/solidity/issues/3452

### CODE REVIEW

File: wrapper.js [hunk 1]
```diff
@@ -6,71 +6,78 @@ var https = require('https');
...
 
 function setupMethods (soljson) {
-  var compileJSON = soljson.cwrap('compileJSON', 'string', ['string', 'number']);
-  var compileJSONMulti = null;
-  if ('_compileJSONMulti' in soljson) {
-    compileJSONMulti = soljson.cwrap('compileJSONMulti', 'string', ['string', 'number']);
-  }
-  var compileJSONCallback = null;
-  var compileStandard = null;
-  if (('_compileJSONCallback' in soljson) || ('_compileStandard' in soljson) || ('_solidity_compile' in soljson)) {
```
Comment (chriseth) [inline wrapper.js]: When did we use `_solidity_compile` and where is it now?

Comment (axic) [reply wrapper.js]: It is the new API, present in 0.5.0.

Comment (axic) [reply wrapper.js]: Added here https://github.com/ethereum/solidity/pull/4366

Comment (chriseth) [reply wrapper.js]: ok, but where is the function wrapper generated after the PR?

Comment (axic) [reply wrapper.js]: Line 78. It is an alias to `compileStandard`, so if it is found, takes precedence for `compileStandard`.

Comment (axic) [reply wrapper.js]: Easier to review this PR commit-by-commit with whitespace changes disabled.

**Pull request merged**